ROVU/Core/src/utility/Main.java

```diff
@@ -0,0 +1,112 @@
+package utility;
+import java.awt.Color;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import controller.RobotController;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+import project.AbstractSimulatorMonitor;
+import project.Point;
+import robot.RobotHandler;
+import simbad.sim.AbstractWall;
+import simbad.sim.Boundary;
+import simbad.sim.EnvironmentDescription;
+import simbad.sim.HorizontalBoundary;
+import simbad.sim.HorizontalWall;
+import simbad.sim.VerticalBoundary;
+import simbad.sim.VerticalWall;
+import view.ROVUView;
+
+@SuppressWarnings("unused")
+public class Main extends Application{
+	
+	static RobotHandler robot1;
+	static RobotHandler robot2;
+	static RobotHandler robot3;
+	static RobotHandler robot4;
+	
+	 Stage stage = new Stage();
+	    public static Scene scene;
+	    
+	@SuppressWarnings("unused")
+	  public static void getScene(Parent root, Stage primaryStage){
+        Rectangle2D screenBounds = Screen.getPrimary().getVisualBounds();
+        scene = new Scene(root, screenBounds.getHeight() + 200, screenBounds.getHeight());
+        primaryStage.setTitle("ROVU view");
+        primaryStage.setX(0);
+        primaryStage.setY(10);
+        primaryStage.setScene(scene);
+        primaryStage.show();
+
+        primaryStage.setOnCloseRequest(e -> {
+            e.consume();
+          //  closeProgram(primaryStage);
+            //TeacherController.alert.close();
+        });
+    }	
+	
+	
+	
+	public static void main(String[] args)  throws InterruptedException {
+
+		
+		    ROVUView exp = new ROVUView();
+	
+	    /**
+	     * Method to define the application window's characteristics. And to close the window when asked.
+	     *
+	     * @param root
+	     * @param primaryStage
+	     *
+	     */
+	  
```

Comment:
This comment seems like it belongs above the method `getScene`, so it shouldn't reside within this method.